requirements.txt

```diff
@@ -11,3 +11,4 @@ pysyncobj>=0.3.8
 cryptography>=1.4
 psutil>=2.0.0
 ydiff>=1.2.0
+python-json-logger>=2.0.7
```

Comment:
That's a very hard requirement.
For example Ubuntu 22.04 provides 2.0.1 and Ubuntu 20.04 only 0.1.11.

Reply:
I use some feature of python-json-logging that is introduced in 2.0.2, such as renaming json keys and adding static fields to log. Previous versions also work but without these features. Should I use 2.0.2 as requirement or use an older version but mention in doc that some features won't work?

Reply:
Oh... Even Debian Unstable has only 2.0.1 :(
This is yet another reason why it should be optional.
It seems to better stick with the latest (2.0.7) in this case.